source/Calamari/Scripts/Octopus.Features.IISWebSite_BeforePostDeploy.ps1

```diff
@@ -481,8 +481,9 @@ if ($deployAsWebSite)
 		}
 
 		Write-Host "Finding SSL certificate with thumbprint $sslCertificateThumbprint"
-	
-		$certificate = Get-ChildItem Cert:\LocalMachine -Recurse | Where-Object { $_.Thumbprint -eq $sslCertificateThumbprint -and $_.HasPrivateKey -eq $true } | Select-Object -first 1
+
+		$accessibleCertificates = Get-ChildItem Cert:\LocalMachine | Foreach-Object { Get-ChildItem "Cert:\LocalMachine\$($_.Name)" -ErrorAction Ignore }
```

Comment:
This may still print an error message for each inaccessible certificate store. 

- [ ] Verify that the script continues even if some (ignored) error messages are printed

Reply:
Confirmed via `DeployWebPackageToIisFixture` that, when the inner `Get-ChildItem` call refers to something that doesn't exist, script execution continues (e.g. to the following `throw "Could not find certificate under..."` statement), so this change does appear to achieve the desired "skipping" effect.